modules/standard/Regex.chpl

```diff
@@ -455,43 +460,12 @@ class BadRegexError : Error {
                           Refer to https://github.com/google/re2/blob/master/re2/re2.h
                           for more details about error codes.
  */
+deprecated "'Regex.compile' is deprecated. Please use 'new regex' instead."
```

Comment:
@lydia-duncan -- do you want me to change this message to align it with `openfd`'s deprecation message. ie., `Please use regex initializer instead` ?

Reply:
Nah, I think it's fine as is.  Thanks for checking, though!